CONTRIBUTING.md

```diff
@@ -0,0 +1,31 @@
+# Contributing
+
+## How to Contribute
+
+We welcome contributions in the form of issues or pull requests!
+
+We want this to be a place where all are welcome to discuss and contribute, so please note that this project is released with a Contributor Code of Conduct. By participating in this project you agree to abide by its terms. Find the code of conduct in the ``CONDUCT.md`` file on GitHub.
+
+If you have a problem using Autofocus or see a possible improvement, open an issue in the GitHub issue tracker. Please be as specific as you can.
```

Comment:
should this be `autofocus` (inline code, lowercase)?

Also I think `open an issue` should be a link to https://github.com/uptake/autofocus/issues/new